examples/02-Merlin-Models-and-NVTabular-applying-to-your-own-dataset.ipynb

```diff
@@ -0,0 +1,698 @@
+{
```

Comment:
<p>"if it is not available on the disk." -&gt; if it is not already downloaded/cached locally.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/253/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>